README.md

```diff
@@ -1,2 +1,88 @@
 # nordicsmartgovernment
 A reference implementation of managed business transactions
+
+# Get started
+
+## Update Linux
+```
+sudo apt update
+sudo apt-get update
+sudo apt-get upgrade
+```
+
+## Install java, git, maven, docker og docker-compose
+
+##### For Linux
+```
+sudo apt-get install default-jdk
+sudo apt install git
+sudo apt install maven
+sudo apt install docker.io
+sudo apt install docker-compose
+```
+
+##### For Windows
+Git for Windows - https://gitforwindows.org/
+
+Apache Maven - http://maven.apache.org/download.cgi
+
+Docker for Windows - https://hub.docker.com/editions/community/docker-ce-desktop-windows
+
+## Steps only be necessary for Linux
+
+##### Configure Docker to start on Boot
+```
+systemctl start docker
+systemctl enable docker
+```
+
+##### Enable executing Docker and Maven without sudo
+```
+sudo usermod -aG docker ${USER}
```

Comment:
How does this differ from `sudo adduser ${USER} docker`, which I feel is much more easy to understand

Reply:
Har ikke brukt adduser før, men ser på blogosfæren at de begge to har akkurat samme funksjon. Endrer til det mer lettleselige

Reply:
1304d8e